app/src/main/java/org/eyeseetea/malariacare/data/database/iomodules/dhis/importer/PullController.java

```diff
@@ -93,16 +88,23 @@ public void postFinish() {
 
     @Override
     public void pull(final PullFilters filters, final IPullControllerCallback callback) {
```

Comment:
@idelcano can you refactor this method extracting logic to new methods ExexuteDemoPull and ExecuteRemotePull for have the code more readable as UserAccountRepository?